class WrongCallbackException(Exception):
    pass

class NotInitedException(Exception):
    pass
